shared/output/gapic/templates/garbage/lib/so/much/trash/garbage_service/client.rb

```diff
@@ -457,6 +457,8 @@ def get_repeated_garbage request, options = nil
           #     Time limit for this garbage. If not defined, the garbage endures forever.
           #   @param enum [ENUM(GarbageEnum)]
           #     The type of this garbage.
+          #   @param amap [So::Much::Trash::TypicalGarbage::AmapEntry | Hash]
```

Comment:
I would have expected that given `map :amap, :string, :string, 14` above, the proto type here would be `So::Much::Trash::TypicalGarbage::Amap` or an array of `So::Much::Trash::TypicalGarbage::AmapEntry`, not a single `So::Much::Trash::TypicalGarbage::AmapEntry`. Is there something I am missing?

Reply:
You're right, that type should be an array of AmapEntry since it's repeated underneath. I'll look into why it's not catching that.

Reply:
Well, we have a bigger problem. None of the repeated fields anywhere are documented as `Array`. They're all just documented as the base type. I guess we're fixing more bugs...

Reply:
okay, I've now included a fix for this. It exposed that a _lot_ of the types in our yardocs were incorrect, as indicated by the changes to generated files in this PR.